src/components/organisms/AdminVenueView/components/RunTabUsers/RunTabUsers.tsx

```diff
@@ -39,6 +39,10 @@ export const RunTabUsers: React.FC<RunTabSidebarProps> = ({ space }) => {
     [owners]
   );
 
+  if (!space) {
+    return null;
+  }
+
```

Comment:
Getting rid of these ones isn't. 

Sadly I had to revert few things to mostly how they were for simplicity while the redesign is going on. 

Considering every React component that has children components is a HOC, introducing specifically named ones like `WithPermissions` and `LoginRestricted` reduces this kind of logic. 

And yes, we can eliminate these minor regressions in a next PR with something like a `RunTabUsersHoc` that has all the logic in it, not a `compose`-d one

Although, I think a more generic one like `<LoadingScreen while={ aListOfValues } />` would solve most of these issues as well. Hopefully that generic one can be the still experimental `<Suspense/>`  (https://reactjs.org/docs/concurrent-mode-suspense.html) from React if it becomes stable and/or we create something similar enough that it can be drop-in replacement.